itou/templates/dashboard/api_token.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layout/content_small.html" %}
+{% load bootstrap4 %}
+
+{% block title %}Mes notifications{{ block.super }}{% endblock %}
+
+{% block content %}
+    <h1 class="h1-hero-c1">Mon Token d'API</h1>
+
+    <div class="card-deck mt-3">
+        <div class="card">
+            <div class="card-body">
+                {% if not token %}
+                    <form method="post" action="{% url 'dashboard:api_token' %}">
+                        {% csrf_token %}
+
+                        <div class="col-12">
+                            <div class="form-row align-items-center">
+                                <div class="form-group mb-0 col-12 col-lg order-2">
+                                    <span class="align-middle">Vous n'avez pas encore de token d'API</span>
+                                </div>
+                                <div class="form-group mb-0 col-6 col-lg-auto order-1 order-lg-2">
+                                    <button type="submit" class="btn btn-primary">Créer un token d'API</button>
```

Comment:
C’est le comportement par défaut.
```suggestion
                                    <button class="btn btn-primary">Créer un token d'API</button>
```